features/FEATURE_LWIP/lwip-interface/lwip_stack.h

```diff
@@ -25,7 +25,8 @@ extern "C" {
 #endif
 
 // Access to lwip through the nsapi
-nsapi_error_t mbed_lwip_init(emac_interface_t *emac);
+void mbed_lwip_init();
```

Comment:
Is that mbed C++ style? Don't see it stated anywhere. Checking other driver and platform header files there seem to be far more `()` than `(void)`.